pkg/api/handlers.go

```diff
@@ -279,6 +279,7 @@ func NewQuerierHandler(
 		reg,
 		nil,
 		remoteWriteEnabled,
+		nil,
```

Comment:
note: this sets the available remote write protocols, however Mimir doesn't use Prometheus to handle this. Has own workflow via (gateway), distributor, ingester